toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -550,6 +546,27 @@ os:
     corruptionOption: panic
 ```
 
+The Verity-enabled root filesystem is always mounted as read-only. Its root hash
+and hash tree are computed at build time and verified by systemd during the
+initramfs phase on each boot. When enabling the Verity feature, it is
+recommended to create a writable persistent partition (e.g., /var) for any
+directories that require write access. Critical files and directories, such as
+SSH host keys or logs, can be redirected to the writable partition using
+symlinks or similar methods. See [Verity Filesystem Layout
+Recommendations](./verity.md) for details.
+
+Please note that some services and programs on Azure Linux may require specific
+handling when using Verity. Depending on user needs, there are different
+configuration options that offer tradeoffs between convenience and security.
+Some configurations can be made flexible to allow changes, while others may be
+set as immutable for enhanced security. 
+
+For example, if your image includes the `systemd-growfs-root.service`, which
```

Comment:
Instead of using prose, let's try to make the recommendations more structured. Also, including example configs would be goodness.

For example:

```Markdown
## systemd-growfs-root

This service attempts to resize the root filesystem, which fails since verity makes the root filesystem readonly and a fixed size.

### Solution 1: Do nothing

Since the root filesystem is readonly, the `systemd-growfs-root` service will fail.
However, the only impact will be an error in the boot logs.

### Solution 2: Disable service

Disabling the service removes the error from the boot logs.

`` `yaml
os:
  services:
    disable:
    - systemd-growfs-root
`` `
```

Do something similar for other services (e.g. cloud-init, ssh, auditd, docker, etc.).